database.rules.json

```diff
@@ -1,59 +1,83 @@
 {
```

Comment:
yeah, this is because I did a lot of the rules editing directly in the firebase console, but haddn't updated them them here.